src/standalone/Snow/Snow.jl

```diff
@@ -316,8 +380,12 @@ end
 A helper function which clips the total water flux so that
 snow water equivalent S will not become negative in a timestep Δt.
 """
-function clip_total_snow_water_flux(S, total_water_flux, Δt)
-    if S - total_water_flux * Δt < 0
+function clip_total_snow_water_flux(
+    S::FT,
+    total_water_flux::FT,
+    Δt::FT,
+) where {FT}
+    if S - total_water_flux * Δt < eps(FT)
```

Comment:
Were these changes required?

Reply:
Part of the NaN issues I messaged you about.